File: 1360/solution.py
import datetime

dt_format = '%Y-%m-%d'

class Solution:
    def daysBetweenDates(self, date1: str, date2: str) -> int:
        dt1 = datetime.datetime.strptime( date1, dt_format )
        dt2 = datetime.datetime.strptime( date2, dt_format )
        return ( abs( (dt2-dt1).days ) )

File: 1360/solution.js
/**
 * @param {string} date1
 * @param {string} date2
 * @return {number}
 */
var daysBetweenDates = function(date1, date2) {
  
    dateObj1 = new Date( date1 );
    dateObj2 = new Date( date2 );
        
    return Math.abs(dateObj2-dateObj1) / (1000*60*60*24);

};

console.log( daysBetweenDates( "2020-01-15", "2019-12-31" ) );